build.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
 	implementation 'org.jsoup:jsoup:1.13.1'
 	implementation 'org.codejive:java-properties:0.0.3'
 
+	implementation 'org.fusesource.jansi:jansi:2.4.0'
+	implementation 'info.picocli:picocli-jansi-graalvm:1.2.0'
```

Comment:
why is it this is needed on top of jansi itself ?

Reply:
This second dependency is there to make sure that it also works in GraalVM. But I haven't tested this and since Jansi 2 also added some GraalVM support, it is possible that this dependency isn't required.